docs/src/Button.doc.js

```diff
@@ -9,281 +9,287 @@ import PageHeader from './components/PageHeader.js';
 const cards: Array<Node> = [];
 const card = c => cards.push(c);
 
+const buildDescription = (lines: Array<string>): Node => (
+  <Stack gap={1}>
+    {lines.map((line, idx) => (
+      <Text key={idx} color="gray">
+        {line}
+      </Text>
+    ))}
+  </Stack>
+);
+
 card(
   <PageHeader
     name="Button"
-    description="
-A form component that should be used to make something happen on the same page (i.e. open a modal).
-You are able to specify the color, type, and width of buttons to change their appearance."
+    description="Buttons allow users to take actions, and make choices, with a single click. They are typically found in forms, dialog, and toolbars. Some buttons are specialized for particular tasks, such as navigation or presenting menus."
   />
 );
 
 card(
   <PropTable
     props={[
+      {
+        name: 'accessibilityLabel',
+        type: 'string',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
+          'Use IconButton if you only need a button with an icon and no text.',
+          'Optional with button-type and submit-type buttons.',
+          'Accessibility: It populates aria-label.  Screen readers read the `accessibilityLabel` prop, if present, instead of `text`.',
+        ]),
+        href: 'accessibilityI',
```

Comment:
@AlbertCarreras instead, can we be more descriptive and link to `accessibilityLabel` instead?

Same for `accessibilityII`, which could be something like `accessibility` as the ID and `Accessibility: controls & expanded` as the title.